docs/BuildingForAndroidTV.md

```diff
@@ -0,0 +1,96 @@
+---
+id: building-for-android-tv
+title: Building For Android TV
+layout: docs
+category: Guides (Android)
+permalink: docs/building-for-android-tv.html
+banner: ejected
+next: android-building-from-source
+previous: signed-apk-android
+---
+
+## Build changes
+
+- *Native layer*: To run React Native project on Android TV make sure to make the following changes to `AndroidManifest.xml`
+
+```xml
+  <!-- Add custom banner image to display as Android TV launcher icon -->
+ <application
+  ...
+  android:banner="@drawable/tv_banner"
+  >
+    ...
+    <intent-filter>
+      ...
+      <!-- Needed to properly create a launch intent when running on Android TV -->
+      <category android:name="android.intent.category.LEANBACK_LAUNCHER"/>
+    </intent-filter>
+    ...
+  </application>
+```
+
+- *JavaScript layer*: Support for Android TV has been added to `Platform.android.js`.  You can check whether code is running on Android TV by doing
+
+```js
+var Platform = require('Platform');
+var running_on_android_tv = Platform.isTV;
+```
+
+## Code changes
+
+- *Access to touchable controls*: When running on Android TV the Android framework will automatically apply a directional navigation scheme based on relative position of focusable elements in your views. The `Touchable` mixin has code added to detect focus changes and use existing methods to style the components properly and initiate the proper actions when the view is selected using the TV remote, so `TouchableHighlight`, `TouchableOpacity` and `TouchableNativeFeedback` will "just work".  In particular:
```

Comment:
I don't think there is a point in duplicating this information. Maybe we can add a section "Building for TV devices" and move most of this stuff there. We can split the section re installation by platform in a way its done here: http://facebook.github.io/react-native/docs/running-on-device.html